src/app/component/user/components/profile/edit-profile/social-networks/social-networks.component.html

```diff
@@ -0,0 +1,22 @@
+<ul>
+    <li *ngFor="let socialNetwork of socialNetworks">
+        <div class="link">
+            <a href="{{socialNetwork.link}}"><img src="{{socialNetwork.icon}}" alt=""></a>
```

Comment:
Move img tag into a separate line for better code readability

Reply:
fixed